### PR TITLE
manual update

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Issue
closes https://github.com/networkservicemesh/cmd-forwarder-vpp/pull/449

### CODE REVIEW

File: internal/tests/suite_memif_test.go [hunk 1]
```diff
@@ -82,8 +82,7 @@ func newMemifVerifiableEndpoint(ctx context.Context,
 						up.NewServer(ctx, vppConn),
 						connectioncontext.NewServer(vppConn),
 						tag.NewServer(ctx, vppConn),
-						memif.NewServer(vppConn),
-						sendfd.NewServer(),
```
Comment (denis-tingaikin) [inline internal/tests/suite_memif_test.go:86]: Why is this removed?

Comment (Mixaster995) [reply internal/tests/suite_memif_test.go:86]: @denis-tingaikin `sendfd` server already present in this endpoint's chain(at the beginning)
